test/torch/tensors/test_autograd.py

```diff
@@ -541,3 +541,73 @@ def test_get_float_prec_on_autograd_tensor(workers):
     x2 = x.fix_precision()
     x2 = x2.share(bob, alice, crypto_provider=james, requires_grad=True)
     assert (x2.get().float_precision() == x).all()
+
+
+def test_serialize_deserialize_autograd_tensor(workers):
+    # let's try to send an autogradTensor to a remote location and get it back
+    alice = workers["alice"]
+    random_tensor = torch.randn(5, 3)
+    remote_tensor = random_tensor.send(alice, local_autograd=True)
+    local_tensor = remote_tensor.get()
```

Comment:
This tests doesn't work: there are no autogradtensor created here because:
- you need to set requires_grad to true on tensors
- send(..., local_autograd=True) creates an autogradtensor locally, you would need to send this object to another worker to have the expected behaviour

I would suggest check types of your local and remote objects in the tests to avoid this and to improve readability :)

Reply:
Alright, so would it be fine  to have:

`random_tensor = torch.randn(5, 3)` ?

Reply:
Better to not use randn I think to be more elegant, try `tensor = torch.tensor([[3.0, 2], [-1, 2]], requires_grad=True)` for example